packages/candid/src/types.ts

```diff
@@ -4,4 +4,4 @@ export interface JsonArray extends Array<JsonValue> {}
 
 export interface JsonObject extends Record<string, JsonValue> {}
 
-export type JsonValue = boolean | string | number | JsonArray | JsonObject;
+export type JsonValue = boolean | string | number | bigint | JsonArray | JsonObject;
```

Comment:
Not sure about the context, but out of curiosity: is this spec-compliant? Or could it cause issues and lead to confusion, since `bigint` is not a valid JSON value?